Refactoring (#6)
IYP-Programer-Yeah/PGen: README.md

A **simple to use** graphical LL(1) parser generator

## Getting Started

> For run this project on your own local machine or server you should install java and maven.

Reviewer: ```suggestion
This project uses Java, hence to run it, you machine should be Java enabled. To compile this project you are going to need maven, however, if you don't want to compile from source, there are releases you can use.
```

You can download jar which is released and use it without compiling it with maven

### Prerequisites

> For installing dependencies for this project do this instructions.

Reviewer: ```suggestion
Following prerequisites are required, depending on your usage.
```

> Install java

Reviewer: ```suggestion
#### Java

To execute this project you are going to need Java JDK, which you can install via the following command, if you are on a Linux machine.

```

```
sudo apt install openjdk-8-jdk
```

> Install maven

Reviewer: ```suggestion
#### Maven

You are going to need Maven only if you are compiling from source. You can install Maven using the following command if you are on a Linux machine.

```

```
sudo apt install maven
```

> ### Installing

Reviewer: ```suggestion

### Compilation

You can compile the code via the following command.
```

> You can compile PGen with below command:

Reviewer: ```suggestion
```

Reviewer: This need to be applied, the line above this addresses this.

```
mvn package
```

> After that, You will see this: `target/PGen-*.*-jar-with-dependencies.jar`. Run that with bellow command

Reviewer: ```suggestion

The result of compilation is a `.jar` file, that you can find in the `target` director. The file is going to be named `PGen-*.*-jar-with-dependencies.jar` where the `*.*` is going to be the version of the binary.
```

Reviewer: `where the `*.*` is going to be the version of the binary.` Is this part correct about the naming? if not modify accordingly.

> ```

Reviewer: ```suggestion

### Execution

You can run the executable with the following command.

```
```
